src/database.ts

```diff
@@ -15,23 +16,30 @@ export class Database {
 
   async connectAndSync() {
     // create a volume-tracker DB if it doesn't exist already
-    const connectionStringParts = DATABASE_URL.split('\/');
-    const connectionStringDBName = connectionStringParts[connectionStringParts.length - 1];
+    const connectionStringParts = DATABASE_URL.split('/');
+    const connectionStringDBName =
+      connectionStringParts[connectionStringParts.length - 1];
     if (connectionStringDBName !== DATABASE_NAME) {
-      logger.info('Database name in connection string is different than expected');
+      logger.info(
+        'Database name in connection string is different than expected',
+      );
       const client = new Client({ connectionString: DATABASE_URL });
       await client.connect();
       try {
         await client.query(`CREATE DATABASE ${DATABASE_NAME};`);
-        logger.info('Created expected database')
+        logger.info('Created expected database');
       } catch (e) {
         if (e.code && e.code === '42P04') {
           logger.info('Expected database already exists');
         } else {
           logger.error('Failed creating expected database', e);
         }
       }
-      connectionStringParts.splice(connectionStringParts.length - 1, 1, DATABASE_NAME);
+      connectionStringParts.splice(
+        connectionStringParts.length - 1,
+        1,
+        DATABASE_NAME,
+      );
```

Comment:
This is the same as `connectionStringParts[connectionStringParts.length - 1] = DATABASE_NAME`
Just saying

[Comment](https://github.com/paraswap/paraswap-volume-tracker/pull/15/files#r842524409) applies